pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -86,13 +86,13 @@ var _ CIDRAllocator = (*cloudCIDRAllocator)(nil)
 // NewCloudCIDRAllocator creates a new cloud CIDR allocator.
 func NewCloudCIDRAllocator(client clientset.Interface, cloud cloudprovider.Interface, nodeInformer informers.NodeInformer) (CIDRAllocator, error) {
 	if client == nil {
-		klog.Fatalf("kubeClient is nil when starting NodeController")
+		return nil, fmt.Errorf("kubeClient is nil when starting cloud CIDR allocator")
```

Comment:
nit: When there's no need for formatting, use `errors.New`. Also applies below.

Reply:
Looks like `errors` now gets imported too often:

```
/home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go:34:2: errors redeclared in this block
```

Reply:
Yeah, I have updated it.